internal/client.go

```diff
@@ -666,8 +666,25 @@ func (e GenericOpenAPIError) Model() interface{} {
 	return e.model
 }
 
+// WithError ,WithBody and WithModel functions for generic OpenAPIError
```

Comment:
why do you need this functions? are the existing constructors not enough?

Reply:
resolved